workflow/rocoto/gfs_tasks.py

```diff
@@ -2680,7 +2680,7 @@ def _get_ecengroups():
                      'maxtries': '&MAXTRIES;'
                      }
 
-        metatask_dict = {'task_name': f'{self.run}ecmn',
+        metatask_dict = {'task_name': f'{self.run}_ecmn',
```

Comment:
Not something necessarily for this PR, but it would be nice to have simpler metatask names, like `f'{self.run}_ecen'`.

Reply:
Improving the task names is planned as a next step.